src/sw.js

```diff
@@ -141,19 +128,53 @@ const navigationNormalizationPlugin = {
 
 precacheAndRoute(self.__WB_MANIFEST);
 
+// HTML caching strategy
+const htmlStrategy = new StaleWhileRevalidate({
+  cacheName: 'pages-cache',
+  plugins: [navigationNormalizationPlugin],
```

Comment:
Question: why remove the `CacheableResponsePlugin` bit here? I believe I had included that to ensure that I only cache HTML here, and that 302s don't get cached (especially as the SW is serving a bunch of dynamic 302s)

Reply:
Oh, sorry. I forgot that dynamic `302`s shouldn't be cached.